content/en/user/run-your-own.md

```diff
@@ -23,18 +25,18 @@ Here is what you need:
 
 - A **domain name**. This is how you and others will access your server and how you and your users will be identified on the network.
 
-  **How to get**: Namecheap, Gandi, any of the infinite number of domain name registrars. Comes with a yearly cost that varies depending on domain name choice.
-- A **VPS**. Something that will run the Mastodon code that will always be connected to the internet.
+  **How to get**: Namecheap, Gandi, Hover, or any of the large number of domain name registrars. A domain comes with an annual cost, that varies depending on the choice of domain name.
```

Comment:
`any of the large number of` -> `one of the many existing`

`cost,` -> `cost` or otherwise `that` -> `which`